### PR TITLE
git, terraform, puppet: updated to the latest runtime v2 changes

### PR DESCRIPTION
**Pull request by ibodrov**
- update `git` and `puppet` tasks to to work with the latest addition to the runtime v2 SDK - `TaskResult`;
- adds `runtime: concord-v2` support to the Terraform plugin;

Requires https://github.com/walmartlabs/concord/pull/61 and https://github.com/walmartlabs/concord/pull/65
Marking this PR as "blocked" until a new version of Concord is released.

### CODE REVIEW

**Comment by ibodrov** [comment]
@walmartlabs/concord Just to make this to compile, I'm going to port other tasks to the new version of the SDK. :-)

File: tasks/git/src/test/java/com/walmartlabs/concord/plugins/git/v2/GitTaskV2Test.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package com.walmartlabs.concord.plugins.git.v2;
```
Comment (benbroadaway) [inline tasks/git/src/test/java/com/walmartlabs/concord/plugins/git/v2/GitTaskV2Test.java:1]: License?

File: tasks/terraform/src/main/java/com/walmartlabs/concord/plugins/terraform/actions/TerraformActionResult.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
+package com.walmartlabs.concord.plugins.terraform.actions;
```
Comment (benbroadaway) [inline tasks/terraform/src/main/java/com/walmartlabs/concord/plugins/terraform/actions/TerraformActionResult.java:1]: License?

File: tasks/git/src/test/java/com/walmartlabs/concord/plugins/git/v2/GitTaskV2Test.java (hunk 1 above)
Comment (ibodrov) [reply tasks/git/src/test/java/com/walmartlabs/concord/plugins/git/v2/GitTaskV2Test.java:1]: Heck, where did the plugin go?

**Review by brig — APPROVED** [review]

**Pull request merged**